pettingzoo/classic/rlcard_envs/leduc_holdem.py

```diff
@@ -93,8 +94,13 @@
 
 
 def env(**kwargs):
-    env = raw_env(**kwargs)
-    env = wrappers.CaptureStdoutWrapper(env)
+    render_mode = kwargs.get("render_mode")
+    if render_mode == "ansi":
+        kwargs["render_mode"] = "human"
+        env = raw_env(**kwargs)
+        env = wrappers.CaptureStdoutWrapper(env)
+    else:
+        env = raw_env(**kwargs)
```

Comment:
This seems like a better way of doing that funky thing where line 37 won't run earlier.